src/views/Artist.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import GridContainer from '../components/Grid/GridContainer';
+import GridItem from '../components/Grid/GridItem';
+import noImageFound from '../assets/img/no_image_found.png';
+import { numberWithCommas } from '../services/utils';
+import {
+  getArtist,
+  getArtistsAlbums,
+  getArtistsTopTracks,
+  getArtistsRelatedArtists
+} from '../actions/artistActions';
+import SongTable from '../components/SongTable';
+import SingleLineGridList from '../components/Grid/GridList';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  coverImage: {
+    backgroundPosition: '50%',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    margin: '0 auto 1em',
+    width: '100%',
+    height: '300px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'flex-end'
+  },
+  text: {
+    background: 'linear-gradient(60deg, #26c6da, #00acc1)',
+    boxShadow:
+      '0 12px 20px -10px rgba(0, 188, 212, 0.28), 0 4px 20px 0px rgba(0, 0, 0, 0.12), 0 7px 8px -5px rgba(0, 188, 212, 0.2)',
+    borderRadius: '3px',
+    color: 'white'
+  },
+  artistName: {
+    padding: '0 10px'
+  },
+  followers: {
+    padding: '1rem'
+  }
+};
+
+class Artist extends Component {
+  constructor(props) {
+    super(props);
+    this.artistId = this.props.match.params.id;
+  }
+
+  componentDidMount() {
+    this.getArtist(this.artistId);
+    this.getArtistsAlbums(this.artistId);
+    this.getArtistsTopTracks(this.artistId);
+    this.getArtistsRelatedArtists(this.artistId);
+  }
+
+  getArtist = id => {
+    this.props.getArtist(id);
+  };
+  getArtistsAlbums = id => {
+    this.props.getArtistsAlbums(id);
+  };
+  getArtistsTopTracks = id => {
+    this.props.getArtistsTopTracks(id);
+  };
+  getArtistsRelatedArtists = id => {
+    this.props.getArtistsRelatedArtists(id);
+  };
+
+  getWiderImage(images) {
```

Comment:
Is this duplicated?

Reply:
the function `getWiderImage`? I don't think so